pgx/src/heap_tuple/mod.rs

```diff
@@ -0,0 +1,337 @@
+//! Provides a safe interface to Postgres `HeapTuple` objects.
```

Comment:
This can be made into `pgx/src/heap_tuple.rs` perhaps?

Reply:
ehh... yah, I suppose.  I think it's going to be self-contained now